pkg/types/interfacetests/chain_reader_interface_tests.go

```diff
@@ -16,6 +16,8 @@ import (
 
 type ChainReaderInterfaceTester[T TestingT[T]] interface {
 	BasicTester[T]
+	StartChainReader(t T)
```

Comment:
Didn't find a way to make the tests pass without doing this breaking change. Open to ideas on how to prevent the multi repo PR!